### PR TITLE
Auto rerun once for top flaky packages targets

### PR DESCRIPTION
**Pull request by keyonghan**
Part of https://github.com/flutter/flutter/issues/135449

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -403,6 +404,7 @@ targets:
       channel: stable
       version_file: flutter_stable.version
       package_sharding: "--shardIndex 1 --shardCount 6"
+      presubmit_max_attempts: "2"
```
Comment (godofredoc) [inline .ci.yaml:407]: Can we add a tracking bug?

**Review by stuartmorgan — COMMENTED** [review]
This will not really work as intended; the shards aren't stable since the packages run on a given shard depend on the packages changed in the PR.

Also, the kind of test that is flaking here already has its own internal retry logic. The problem is that a test in this shard has recently become extremely flaky; we're trying to isolate which one to disable it so that we don't have to turn off the entire plugin's tests. I don't think adding a second (less efficient) layer of retries is a good idea here.

**Comment by keyonghan** [comment]
> This will not really work as intended; the shards aren't stable since the packages run on a given shard depend on the packages changed in the PR.
> 
> Also, the kind of test that is flaking here already has its own internal retry logic. The problem is that a test in this shard (when running all packages) has recently become extremely flaky; we're trying to isolate which one to disable it so that we don't have to turn off the entire plugin's tests. I don't think adding a second (less efficient) layer of retries is a good idea here.

Proposing this change mainly due to the annoying flake that happens on the `dependbot` dependency bumping PRs. For example https://github.com/flutter/packages/pull/4988. Infra team has to manually keep rerunning the failed/flaky targets and adding the autosubmit label until they pass.

Is there any bug tracking the fix/skip of the specific flaky tests in these flaky shards?

**Comment by stuartmorgan** [comment]
> Is there any bug tracking the fix/skip of the specific flaky tests in these flaky shards?

https://github.com/flutter/flutter/issues/130986 is tracking the current bad flake.

**Pull request closed without merging**